pkg/controller/kieapp/constants/constants.go

```diff
@@ -19,7 +19,7 @@ const (
 	// ProductVersion default version
 	ProductVersion = "7.4"
 	// ImageStreamTag default tag name for the ImageStreams
-	ImageStreamTag = "1.1"
+	ImageStreamTag = "1.0"
```

Comment:
I think this is meant to be at 1.1 for master, and given that the change will be cherry-picked to the 7.4.0 branch, you should still get a 1.0 tag in there. @tchughesiv ?

Reply:
yes @bmozaffa this should remain at `1.1` for master. the cherry-pick will take care of versioning for builds.

Reply:
@spolti please remove this change from your PR